README.md

```diff
@@ -0,0 +1,381 @@
+# Tariff Interpreter
+
+This Interpreter is a reference implementation for computing a JSON formatted bike-sharing tariff model. The tariff
+model is designed to be able to represent as many types of tariffs as possible and to have a modular structure. Thus, it
+is possible to calculate as many tariffs as possible with a single tariff interpreter and at the same time ensure a
+uniform format. In addition, a kind of receipt is issued at the end thanks to a flexible slot system. Thus, the
+composition of the travel costs is broken down as transparently as possible for the end customer.
```

Comment:
I would like to remove the 'as possible'. I think we can state that it's broken down transparently, or do we have any non-transparency?